rs/sns/governance/src/governance.rs

```diff
@@ -3635,9 +3794,16 @@ impl Governance {
         // not (directly or indirectly) voted yet). That is, once a neuron is swayed,
         // its vote is "locked in". IOW, swaying is "monotonic".
         while !induction_votes.is_empty() {
+            #[derive(Eq, Ord, PartialEq, PartialOrd)]
+            enum FollowingType {
```

Comment:
This info is most likely not necessary 

Try to rewrite without keeping track of the edge type